modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/UniqueComposedKey.tsx

```diff
@@ -39,10 +68,13 @@ export function UniqueComposedKey({
 	const [buildScreenItems, setBuildScreenItems] = useState<
 		TBuilderScreenItem[]
 	>([]);
-	const [multipleSelectOptions, setMultipleSelectOptions] = useState<IItem[]>(
-		[]
-	);
-
+	const [
+		modalSelectObjectFieldsItems,
+		setModalSelectObjectFieldsItems,
+	] = useState<ModalSelectObjectFieldItemProps[]>([]);
```

Comment:
@igor-franca To follow the pattern of other states, please rename it to `ModalSelectObjectFieldItem`.